lldb/source/API/SBValue.cpp

```diff
@@ -1233,7 +1233,10 @@ bool SBValue::GetDescription(SBStream &description) {
     DumpValueObjectOptions options;
     options.SetUseDynamicType(m_opaque_sp->GetUseDynamic());
     options.SetUseSyntheticValue(m_opaque_sp->GetUseSynthetic());
-    value_sp->Dump(strm, options);
+    if (llvm::Error error = value_sp->Dump(strm, options)) {
+      strm << "error: " << toString(std::move(error));
+      return false;
```

Comment:
Do we call this a bug fix, or keep the original behaviour of always returning true? Given that this is part of the API, whereas the interactive commands are internal.

Reply:
The docs are non-existent, but just from the prototype I'd assume it can return false already: https://lldb.llvm.org/python_api/lldb.SBValue.html#lldb.SBValue.GetDescription

So it might break someone's existing script but not in a totally unexpected way. I'd lean toward this being a bug fix.

Reply:
Yeah this is a drive-by fix where I'm pretty sure that this is the better behavior.
In practice no ObjC object description ever fails completely, so I don't know how to test this.